### PR TITLE
Update Buildbarn components

### PR DESCRIPTION
**Pull request by stagnation**
Update to the latest(ish) Buildbarn, and tinker with the WORKSPACE.
This pins the AWS SDK versions to what bb-storage uses (for its aws backend).

### CODE REVIEW

**Review by EdSchouten — APPROVED** [review]

**Comment by EdSchouten** [comment]
Just made you a maintainer. Feel free to merge when you like it 😀

**Comment by moroten** [comment]
LGTM. Will merge when CI succeeds.

**Pull request merged**
